common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -0,0 +1,31 @@
+﻿#### Local Types ####
```

Comment:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105886144-7a7bfd00-6023-11eb-8a5b-663665a1f709.png)

</details>